blog-posts/felix-interview.md

```diff
@@ -0,0 +1,65 @@
+
+<br>**Klingt so als wärst Du immer lernhungrig und bräuchtest nie eine Pause ...**<br>
+Wenn ich nicht von Jemandem abgelenkt werde (Freundin ❤️), vergesse ich alles um mich herum. Ich vergesse zu essen, trinken, ich bin immer total im Tunnel. Ich überhöre sogar den Pomodoro-Timer, den mir ein Kollege mal dafür empfohlen hat. 
+
+<br>**Aber Wochenenden bekommst Du schon mit, oder? Was machst Du dann?**<br>
+Wenn’s das Wetter erlaubt gehe ich Gleitschirm fliegen, "Paragliden". Generell verbringe ich viel Zeit in der Natur, beim Pilze sammeln oder Bergsteigen. Ansonsten programmieren. Zur Zeit beispielsweise machen viele meiner Kollegen und ich bei “Advent of Code” mit. Das ist ein Adventskalender für Nerds mit täglich zwei spannenden Programmier-Herausforderungen. Challenges, die ich unter der Woche nicht geschafft habe, hole ich dann am Wochenende nach. Und (natürlich) viel Quality Time mit meiner Freundin.
+
+<br>**Wie bist Du auf’s Paragliding gekommen? Gibt es Gemeinsamkeiten oder Gegensätze zu deiner Arbeit?**<br>
+Durch meinen Bruder. Beim Paragliden kann man sich nur mit viel Erfahrung ein konkretes Ziel setzen, aber meistens fliegt man einfach drauf los und schaut wie Wind und Wetter mitmachen. Man kann schon sagen: “Ich versuche zu diesem und jenem Gebirgskamm zu fliegen.”, aber es ist nicht planbar, nicht systematisch. Man ist der Umwelt ausgeliefert, einer plötzlichen Wolke vor der Sonne, einer anderen Windrichtung wenn man ein bisschen höher oder tiefer fliegt. Man ist im Hier und Jetzt und gibt die Kontrolle ab – ein willkommener Unterschied zum Entwickeln, wo man doch im Großen und Ganzen “Herr der Lage” ist.
+
+<br>**Schön beschrieben, danke dafür. Apropos Hier und Jetzt. Was steht heute noch an?**<br>
+Wir sind heute mit einer neuen Version von Satellytes.com live gegangen und haben da noch ein bisschen was zu tun. Außerdem geht einer meiner Kollegen bald für drei Wochen in den Urlaub, deswegen müssen noch zwei Pull Requests von ihm gereviewed und gemerged werden solange er noch da ist – und unbedingt abspülen. Mein Zuhause sieht aktuell aus wie S...
+
+<br>**Zuhause ist ...**<br>
+... wo sich mein Handy automatisch mit dem WLAN verbindet. Klingt bisschen klischeehaft, aber der Spruch bewahrheitet sich immer wieder. Ich nehme mir nur dort die Zeit mich mit dem Internet zu verbinden, wo ich es mag und vielleicht nochmal herkommen möchte.
+
+<br>**Letzte Worte?**<br>
+Ich freue mich auf die nächsten `Math.pow(3.75, x)` Jahre bei Satellytes 🙌
```

Comment:
I get the magic number 3.75 but I don't get the pow 😅 Time is linear shouldn't it be `3.75 * (1 + x)` or such? Help mee

Reply:
I think we can't deny anymore to be nerds 😅